### DISCUSSION TITLE
firmware V1.0.1.49 has great new feature that appears to break some of the variables changing

### DISCUSSION THREAD

**Discussion by dltccf** [vwt12eh8/hassio-ecoflow#80] [General]
I recently got this update from .18 and it has a great new feature called "energy management" which allows you to set the level where AC charging kicks in. It allows you to set what ecoflow calls Backup resource level. above that percent of charge AC charging is off, below it is on. I asked for a feature exactly like this a year ago, so they must have included me in a limited rollout. 

What it allows you to do is you can set it at a low percentage, say 10%, and the Delto Pro will only charge from solar above that percentage. If it gets below the AC charging kicks in as a backup. 

Now however, the ecoflow device has only 63 entries rather than 69. Also the device itself keeps toggling back and forth between Status of "detected" and "cleared" .  What information can I send to this forum to help fix this for when this feature rolls out to everyone else?

**Comment by dltccf** [comment]
![Screenshot from 2023-05-11 11-23-28](https://github.com/vwt12eh8/hassio-ecoflow/assets/131302887/b7bb1535-bb26-4544-b90a-da8ecb3b1b4d)

**Comment by dltccf** [comment]
![Screenshot from 2023-05-11 11-15-43](https://github.com/vwt12eh8/hassio-ecoflow/assets/131302887/50bde069-16b4-4102-a712-9254b10c9df0)

**Comment by dltccf** [comment]
![Screenshot from 2023-05-11 11-12-29](https://github.com/vwt12eh8/hassio-ecoflow/assets/131302887/188fdee4-181b-445b-91ec-0ca2d42507c3)

**Comment by dltccf** [comment]
![Screenshot from 2023-05-11 11-40-58](https://github.com/vwt12eh8/hassio-ecoflow/assets/131302887/3931d261-c0e7-48bd-bf13-db33e942fb89)

**Comment by DarrylGNOLA** [comment]
I have also received the notification for this update recently, but haven't done it yet. I came here first to see if there are any issues. Has anyone else done this and have the same issues? I don't want to loose my monitoring of the MPPT, etc. Did they possibly rename these slightly?
